### ISSUE TITLE
Sound issue

### ISSUE THREAD

**Issue by zedtux** [aroman/freya-on-a-mac#24]
I don't know if it's the right place to open this issue so I try :)

I have installed Freya some months ago and never really used it until today. I've booted it up and upgraded everything. Then I have installed Spotify ... no sound. Looking around on the internet I have found [this post on elementaryos.org](http://elementaryos.org/answers/external-speaker-dont-work-on-mac71) which brings me sound ... but the sound is missing trebles.

Someone know how to solve this issue ?
Where can I search/ask for this ?

**Comment by zedtux** [comment]
I have found [this SO post](http://askubuntu.com/a/72686) which allowed me to improve the sound quality but it's not like on OSX ...

**Comment by sdaitzman** [comment]
What hardware do you have? I've never experienced this issue but that doesn't mean it isn't there :)

**Comment by zedtux** [comment]
I have an iMac later 2013 (iMac14,2) with the following:

    00:00.0 Host bridge: Intel Corporation 4th Gen Core Processor DRAM Controller (rev 06)
    00:01.0 PCI bridge: Intel Corporation Xeon E3-1200 v3/4th Gen Core Processor PCI Express x16 Controller (rev 06)
    00:14.0 USB controller: Intel Corporation 8 Series/C220 Series Chipset Family USB xHCI (rev 05)
    00:16.0 Communication controller: Intel Corporation 8 Series/C220 Series Chipset Family MEI Controller #1 (rev 04)
    00:1b.0 Audio device: Intel Corporation 8 Series/C220 Series Chipset High Definition Audio Controller (rev 05)
    00:1c.0 PCI bridge: Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #1 (rev d5)
    00:1c.2 PCI bridge: Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #3 (rev d5)
    00:1c.3 PCI bridge: Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #4 (rev d5)
    00:1c.4 PCI bridge: Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #5 (rev d5)
    00:1f.0 ISA bridge: Intel Corporation Z87 Express LPC Controller (rev 05)
    00:1f.3 SMBus: Intel Corporation 8 Series/C220 Series Chipset Family SMBus Controller (rev 05)
    01:00.0 VGA compatible controller: NVIDIA Corporation GK104M [GeForce GTX 775M Mac Edition] (rev a1)
    01:00.1 Audio device: NVIDIA Corporation GK104 HDMI Audio Controller (rev a1)
    02:00.0 SATA controller: Samsung Electronics Co Ltd Apple PCIe SSD (rev 01)
    03:00.0 Network controller: Broadcom Corporation BCM4360 802.11ac Wireless Network Adapter (rev 03)
    04:00.0 Ethernet controller: Broadcom Corporation NetXtreme BCM57766 Gigabit Ethernet PCIe (rev 01)
    04:00.1 SD Host controller: Broadcom Corporation BCM57765/57785 SDXC/MMC Card Reader (rev 01)

**Comment by zedtux** [comment]
I have switched to Debian and I have the same issue.

To be more precise the issue is the following:

* When booting the OS and using the keyboard media keys in order to increase / decrease the volume I can hear the little sound but then firing Spotify and starting to play a song make the iMac completely silent
* or sometimes the sound is not working at all
* finally sometimes it works like a charm

Now regarding the trebles issue I think it's just a configuration issue with the equalizer which is much better configured out-of-the-box with OSX than Linux.

**Comment by aroman** [comment]
FWIW, back in the day (~4+ years ago) I used to run elementary/Linux on a 2008 iMac 8,1. I did definitely experience crappy sound — like it was audible, but it clearly sounded worse than under OS X. I spent many hours messing around with various settings (e.g. `alsamixer`), and I did manage to get it a bit better by fiddling with equalizers and setting the various speakers to play at certain volumes relative to each other, but I never did get it as good as OS X.

I don't have that problem with my 13'' Retina Macbook Pro (2014), which is my current machine.

I'm gonna mark this as closed, since I'm fairly certain this issue is at the kernel/hardware level, and thus (unfortunately) effectively out of scope for elementary and also this guide.
